internal/command/test_test.go

```diff
@@ -2333,3 +2335,83 @@ required.
 		t.Errorf("should have deleted all resources on completion but left %v", provider.ResourceString())
 	}
 }
+
+func TestTest_JUnitOutput(t *testing.T) {
+
+	tcs := map[string]struct {
```

Comment:
There are probably other test cases to add, but that might be in a separate PR where I look into the remaining FIXMEs ([example](https://github.com/hashicorp/terraform/blob/43f31c43b0a704a8ee9e58cfb7af539a92b80a4d/internal/command/views/test.go#L928-L930))

Reply:
There isn't an example of a skipped test, though I think that'd be achievable as a test within the artifact package directly (no sources would be needed when making a file with only skipped tests)

Reply:
Addressed in [d01122a](https://github.com/hashicorp/terraform/pull/36304/commits/d01122a275a8c41bbe0f8efa9e663e066a3562b8)